web-admin/src/components/projects/use-project.ts

```diff
@@ -0,0 +1,30 @@
+import { createAdminServiceGetProject } from "@rilldata/web-admin/client";
+
+const PollTimeDuringReconcile = 1000;
+const PollTimeDuringError = 5000;
+const PollTimeWhenProjectReady = 60 * 1000;
+
+export function useProject(orgName: string, projName: string) {
```

Comment:
Given the `refetchInterval` logic, this should be `useProjectStatus` and should use a `select` function to filter for the status.

Otherwise, if we were to this `useProject` elsewhere for other project data, then we'd be refetching unnecessarily.

Might want to change the file name to reflect this change, too.

Reply:
We need to refetch project to get the new deployment details. Since a re-deploy could change the runtime url.

Reply:
But each place we're now using this hook is only accessing the `status` field, not the runtime URL. 

Previously, Benjamin had me add a 30 minute refetch for the JWT in `[project]/+layout.svelte`. Should we change that logic?

I think it's good to be specific (re: selectors, refetchIntervals) per QueryObserver and let Tanstack figure out refetching and deduplication for us.